src/frontend/cli_interactive.py

```diff
@@ -0,0 +1,201 @@
+import curses
+from PIL import Image
+from typing import Any
+
+from constants import DEVICE
+from paths import FastStableDiffusionPaths
+from backend.upscale.upscaler import upscale_image
+from backend.models.lcmdiffusion_setting import DiffusionTask
+from backend.upscale.tiled_upscale import generate_upscaled_image
+from frontend.webui.image_variations_ui import generate_image_variations
+
+_batch_count = 1
+
+
+
+def interactive_mode(config, context):
+    print("=============================================")
+    print("Welcome to FastSD CPU Interactive CLI")
+    print("=============================================")
+    print("> 1. Text to Image")
+    print("> 2. Image to Image")
+    print("> 3. Image Variations")
+    print("> 4. EDSR Upscale")
+    print("> 5. SD Upscale")
+    option = int(input("Enter a Diffusion Task number: "))
+    if option not in range(0, 6):
+        print("Wrong Diffusion Task number!")
+        exit()
+
+    edit_settings = input("Edit default generation settings? (y/N): ")
+    if (edit_settings.upper() == 'Y'):
+        interactive_settings(config, context)
+
+    if option == 1:
+        interactive_txt2img(config, context)
+    elif option == 2:
+        interactive_img2img(config, context)
+    elif option == 3:
+        interactive_variations(config, context)
+    elif option == 4:
+        interactive_edsr(config, context)
+    elif option == 5:
+        interactive_sdupscale(config, context)
+
+
+
+def user_value(value_type: type, message: str, default_value: Any) -> Any:
```

Comment:
Use trailing comma at the end of function arguments.
def user_value(value_type: type, 
                    message: str, 
                    default_value: Any,)
Use black code formatter